lib/ar2dto/active_model.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module AR2DTO
+  module ActiveModel
+    def self.included(base)
+      base.include ::ActiveModel::Conversion
+      base.extend ::ActiveModel::Naming
+      base.extend ::ActiveModel::Translation
+      base.extend ClassMethods
+
+      base.class_eval do
```

Comment:
I need to check if putting everything in `base` is better or just having things on the `AR2DTO`ActiveModel` module

Reply:
This way looks good. We want `AR2DTO::DTO` to be the class with all the methods